.test-infra/dataproc/init-actions/flink.sh

```diff
@@ -160,10 +156,10 @@ jobmanager.rpc.address: ${master_hostname}
 jobmanager.heap.mb: ${flink_jobmanager_memory}
 taskmanager.memory.process.size: "${flink_taskmanager_memory} mb"
 taskmanager.memory.jvm-metaspace.size: 512 mb
+taskmanager.memory.task.off-heap.size: 256 mb
```

Comment:
The jobs had issues with direct memory being too low. This setting solved the problem.

Reply:
Sounds good!